client/index.html

```diff
@@ -24,24 +24,59 @@
     <script src="./explore-playlists/explore-playlists-factory.js" charset="utf-8"></script>
 
     <script src="./player/player-ctrl.js" charset="utf-8"></script>
-    <script src="./player/player-factory.js" charset="utf-8"></script>
-
-  </head>
+    <script src="./player/player-factory.js" charset="utf-8"></script>
+  </head>
   <body>
-
-    <nav>
-      <div class="nav-wrapper">
-        <ul id="nav-mobile" class="left hide-on-med-and-down">
-          <li><a href="#/create">Create Playlist</a></li>
-          <li><a href="#/playlist">Single Playlist</a></li>
-          <li><a href="#/explore">Explore Playlists</a></li>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="#">PlaylistApp</a>
+        </div>
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="#">Home</a></li>
+        </ul>
+        <div class="col-sm-3 col-md-3">
+          <form class="navbar-form" role="search">
+            <div class="input-group">
+              <input type="text" class="form-control" placeholder="Search Playlists" name="q">
+            <div class="input-group-btn">
+                <button class="btn btn-default" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+            </div>
+            </div>
+          </form>
+        </div>
+        <ul class="nav navbar-nav navbar-right">
+          <li><a href="#"><span class="glyphicon glyphicon-user"></span>Profile</a></li>
+          <li class="dropdown">
+          <a class="dropdown-toggle" data-togggle="dropdown" href="#">Playlist<span class="caret"></span></a>
+          <ul class="dropdown-menu">
+            <li><a href="#/explore">Explore Playlists</a></li>
+            <li><a href="#/playlist">Single Playlist</a></li>
+            <li><a href="#/create">Create Playlist</a></li>
+          </ul>
+          </li>
+          <li><a href="#">Logout</a></li>
         </ul>
       </div>
     </nav>
-
-
     <div ui-view></div>
 
-    <div id="music-player" ng-include="'./player/player.html'"></div>
+    <div id="music-player">
```

Comment:
Remove line 64-80,
music player is in the player.html, as ng-include in the line u removed 
 <div id="music-player" ng-include="'./player/player.html'"></div>